monai/data/decathlon_datalist.py

```diff
@@ -63,9 +72,12 @@ def _append_paths(base_dir: str, is_segmentation: bool, items: List[Dict]) -> Li
             raise TypeError(f"Every item in items must be a dict but got {type(item).__name__}.")
         for k, v in item.items():
             if k == "image":
-                item[k] = _compute_path(base_dir, v)
+                item[k] = _compute_path(base_dir, v, check_path=False)
             elif is_segmentation and k == "label":
```

Comment:
I believe it is better to remove `is_segmentation` because it might be confusing since "label" are processing either here or in the next if (depending on this flag) and by removing. Also, what if we are doing segmentation but using patch-wise classification, then the labels are still values and not masks.